src/plugins/data/public/search/expressions/es_raw_response.test.ts

```diff
@@ -0,0 +1,827 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { EsRawResponse, esRawResponse } from './es_raw_response';
+
+jest.mock('@kbn/i18n', () => {
+  return {
+    i18n: {
+      translate: (id: string, { defaultMessage }: { defaultMessage: string }) => defaultMessage,
+    },
+  };
+});
+
+describe('esRawResponse', () => {
+  describe('converts aggregations to table', () => {
+    test('simple aggregation response', () => {
+      const response: EsRawResponse = {
+        type: 'es_raw_response',
+        body: {
+          took: 7,
+          timed_out: false,
+          _shards: {
+            total: 7,
+            successful: 7,
+            skipped: 0,
+            failed: 0,
+          },
+          hits: {
+            total: 1977,
+            max_score: 0,
+            hits: [],
+          },
+          aggregations: {
+            '2': {
+              doc_count_error_upper_bound: 0,
+              sum_other_doc_count: 0,
+              buckets: [
+                {
+                  key: 'FEMALE',
+                  doc_count: 1033,
+                },
+                {
+                  key: 'MALE',
+                  doc_count: 944,
+                },
+              ],
+            },
+          },
+        },
+      };
+      const result = esRawResponse.to!.datatable(response, {});
+      expect(result).toMatchSnapshot();
```

Comment:
These snapshot tests are hard to review. Can you assert something specific about the structure of the response here?